docs-src/references/strongly-typed-errors/cause-bad-update.md

```diff
@@ -0,0 +1,18 @@
+---
+id: cause-bad-update
+title: Cause Bad Update
+description: Explanation for Cause Bad Update error message.
+sidebar_label: Cause Bad Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+<!--TODO: add link to Workflow Update page when written -->
+
+This error indicates that a [Workflow Execution](/concepts/what-is-a-workflow-execution) tried to complete before receiving an Update.
```

Comment:
```suggestion
This error indicates that a [Workflow Execution](/concepts/what-is-a-workflow-execution) tried to complete before receiving an update.
```
Lowercase unless, of course, we're gonna claim "Update" as a Temporal-specific term and add it to the Glossary.